CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # CHANGELOG
 
+## 1.12.0
+
+**New**
+
+* Implement new EmojiFilter context option: ignored_ancestor_tags to accept more ignored tags. [#170](https://github.com/jch/html-pipeline/pull/170) @JuanitoFatas
+* Add GitHub flavor Markdown Task List extension [#162](https://github.com/jch/html-pipeline/pull/162) @simeonwillbanks
+* @mention allow for custom regex to identify usernames. [#157](https://github.com/jch/html-pipeline/pull/157) @brittballard
+
+**Changes**
+* Restrict nokogiri to >= 1.4, <= 1.6.5 [#176](https://github.com/jch/html-pipeline/pull/176) @simeonwillbanks
```

Comment:
Do we need to announce that Active Support version also restricted to `>= 2, < 5`?

```
Restrict Active Support to >= 2, < 5 [#180](https://github.com/jch/html-pipeline/pull/180) @jch 
```